content/raw/chat/group_chat.md

```diff
@@ -0,0 +1,50 @@
+# Group chat
+Group chats are meant to be private ad-hoc groups.
+
+# Rules
+
+### Glossary
+Group creator - one who creates new group  
+Member - anyone in the group  
+Admin - group administrator  
+User - Member who is not an Admin  
+
+## Group creation
+Group creator is only allowed to add mutual contacts. This is important, both for privacy and as a spam prevention measure.
+- Group chat MUST contain at least one Member
+- Group chat MUST be composed only of Group creator's mutual contacts
+- Group creator MUST become an Admin of the group
+
+## Group roles
+- Group chat CAN have more than one Admin
+- Only Admin CAN promote User to the Admin role
```

Comment:
The specs have been updated. There can only be one admin.

One of the solutions would be to have admin in tag selector list, but as non-interactive entry.

Reply:
> The specs have been updated. There can only be one admin.
> 
> One of the solutions would be to have admin in tag selector list, but as non-interactive entry.

Ok! So, 2 options:
1. Admin is displayed in tag selector list but as a non-interactive entry.
2. Admin is not displayed in the tag selector list.

Could @John-44 decide your design preference?!

Reply:
@noeliaSD Let's go for option 1 (Admin is displayed in tag selector but they can't be removed).  I've made a note to design a crown icon (or something like that) to indicate who the admin is in the tag.  Also perhaps the Admin should always be the first (left most) tag in the list.

With this spec there are a couple of other changes:

- when a user who *is not an admin* goes to the 'edit group' option, they shouldn't see "x" on any of the tags of members who are already in the group, because only the admin can remove them.  However if a user who is not an admin goes to this view, they can add more users, and they can also remove these users *before they press confirm*.  Once those users are added, a user who is not an admin cannot remove them.

- the 'add/remove users' menu option should be renamed to something like 'add users' for a user who is not the admin of the group